Patterns/English/English-TimeZone.yaml

```diff
@@ -0,0 +1,292 @@
+---
+# Direct UTC Expression
```

Comment:
The generated .ts file is missing. Please add to PR.

Reply:
I didn't see any .ts file from the old pr. Isn't this file used by a tt file?

Reply:
Every change to a yaml file needs to generate new definition files in both .NET and Typescript.
In this PR you've included the new .cs file, but the new .ts file is missing. Probably it was also missing in the old PR.